snap/snapcraft.yaml

```diff
@@ -92,10 +92,11 @@ parts:
       - libdvdread-dev
       - libegl1-mesa-dev
       - libenca-dev
+      - libfdk-aac-dev
       - libgbm-dev
       - libgl1-mesa-dev
       - libguess-dev
-      - libjack-dev
+      # - libjack-dev
```

Comment:
`libjack-dev` does not break the build on my machine when I use `snapcraft cleanbuild`. Can you describe the issues you were having?